src/NHibernate/Driver/SqlServerCeDriver.cs

```diff
@@ -127,13 +146,14 @@ private void AdjustDbParamTypeForLargeObjects(DbParameter dbParam, SqlType sqlTy
 			{
 				dbParamSqlDbTypeProperty.SetValue(dbParam, SqlDbType.NText, null);
 			}
+#endif
 		}
 
 		public override bool SupportsNullEnlistment => false;
 
 		/// <summary>
 		/// <see langword="false"/>. Enlistment is completely disabled when auto-enlistment is disabled.
-		/// <see cref="DbConnection.EnlistTransaction(System.Transactions.Transaction)"/> does nothing in
+		/// <code>DbConnection.EnlistTransaction(System.Transactions.Transaction)</code> does nothing in
```

Comment:
I think this should be reverted. `code` is for block code and what was the harm of `see`?